examples/getting-started/README.md

```diff
@@ -142,22 +115,82 @@ services:
+we start the server with a port offset is because we will be starting another instance of WSO2MI for the composite 
+service to as explained in the [section below](#deploying-the-composite-service-locally), which could lead to a port conflict.)
+
+   **Note -** If you want to start Micro-Integrator with a portOffset=1 you have to pass the parameter 
+   as ```-DportOffset=11```, because Micro-Integrator starts with a default portOffset=10.
+      
+3. You should be able to access the back-end using the following cURL command
    ```
-   $ cd <mi-work-directory>/examples/getting-started/
-   $ docker-compose up -d
+   curl --header "Content-Type: application/json" --request POST  --data '{"store": {"book": [{"author": "Nigel Rees","title": "Sayings of the Century"},{"author": "J. R. R. Tolkien","title": "The Lord of the Rings","isbn": "0-395-19395-8"}]}}' http://localhost:8291/order
    ```
-Now you have successfully deployed 'order-process-backend' and the 'composite-service' services in Docker. (Execute `docker container ls` to see the running containers).
+#### Deploying the composite service locally
 
-Head over to the [Testing](#Testing) section to test your deployment.
+Assume you have followed steps 1-4 in the section 
+[Create the composite service to invoke the back-end.](#2-create-the-composite-service-to-invoke-the-back-end)
+
+1. Copy the `composite-service/composite-service-capp/target/composite-serviceCompositeApplication_1.0.0.car` 
+into `<MI2_HOME>/repository/deployment/server/carbonapps`
+
+2. Start the Micro Integrator instance(MI2).
+
+You have successfully deployed 'order-process-backend' and the 'composite-service' services locally. 
+Now you can test your deployments by following the [Testing](#Testing) section.
 
 ### Kubernetes Deployment
 
 Please follow all the steps in the section [Create the order processing back-end](#1-create-the-order-processing-back-end) and
 [Create the composite service to invoke the back-end.](#2-create-the-composite-service-to-invoke-the-back-end) if you have not already done. In step 3 of
 [Create the composite service to invoke the back-end](#2-create-the-composite-service-to-invoke-the-back-end), you need
-to replace 'order-process-backend' endpoint address in [forwardOrderApi.xml](https://github.com/wso2/micro-integrator/blob/master/examples/getting-started/composite-service/composite-service/src/main/synapse-config/api/forwardOrderApi.xml) file as follows because in docker containers each container has their own localhost.
+to replace 'order-process-backend' endpoint address in 
+[forwardOrderApi.xml](https://github.com/wso2/micro-integrator/blob/master/examples/getting-started/composite-service/composite-service/src/main/synapse-config/api/forwardOrderApi.xml) 
+file as follows because in docker containers each container has its own localhost.
 ```xml
 <endpoint>
     <address uri="http://order-process-be-service:8290/order"/>
```

Comment:
Do we need to change the backend URL for k8s. Isn't it same for both Docker and k8s?

Reply:
If a user changes it to `localhost` in the local deployment, then they need to change it back for the K8s deployment.

Reply:
:+1: